api/src/main/java/org/openmrs/module/mirebalais/hfe/velocity/LrVelocityContextContentProvider.java

```diff
@@ -0,0 +1,66 @@
+package org.openmrs.module.mirebalais.hfe.velocity;
+
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.velocity.VelocityContext;
+import org.openmrs.Concept;
+import org.openmrs.Condition;
+import org.openmrs.Patient;
+import org.openmrs.api.ConceptService;
+import org.openmrs.api.ConditionService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.htmlformentry.FormEntrySession;
+import org.openmrs.module.htmlformentry.velocity.VelocityContextContentProvider;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+
+@Component
+public class LrVelocityContextContentProvider implements VelocityContextContentProvider {
+
+    @Override
+    public void populateContext(FormEntrySession formEntrySession, VelocityContext velocityContext) {
+        Patient patient = formEntrySession.getPatient();
+
+        ConditionService conditionService = Context.getConditionService();
+
+        List<Condition> conditionList = conditionService.getActiveConditions(patient);
+
+        List<Concept> conditionListConcepts = new ArrayList<Concept>();
+        for (Condition condition : conditionList) {
+            if (condition.getCondition().getCoded() != null) {
+                conditionListConcepts.add(condition.getCondition().getCoded());
+            }
+        }
+
+        List<Concept> mentalHealthDiagnosesConcepts = getMentalHealthDiagnoses();
+
+        velocityContext.put("conditionList", conditionList);
+        velocityContext.put("conditionListConcepts", conditionListConcepts);
```

Comment:
for what it's worth, these two condition contexts are generic enough that they could probably live in a separate context provider and shared by multiple implementations (but no particular reason to show it now)